packages/services/src/helpers/term-id-getter.js

```diff
@@ -0,0 +1,21 @@
+import Network from '@aragonone/court-backend-server/build/web3/Network'
+
+async function draftTermIdFor(state) {
+  const court = await Network.getCourt()
+  const currentTerm = await court.currentTerm()
+  const { roundDurations: { commitTerms, revealTerms } } = await court.getConfigAt()
+  if (state == 'revealing') {
+    return currentTerm.sub(commitTerms)
+  }
+  if (state == 'appealing') {
+    return currentTerm.sub(commitTerms).sub(revealTerms) 
+  }
+  if (state == 'commit-reminder') {
+    return currentTerm.sub(commitTerms.add(1).div(2))
+  }
+  if (state == 'reveal-reminder') {
+    return currentTerm.sub(commitTerms).sub(revealTerms.add(1).div(2))
```

Comment:
I think you will need to use `add(bn(1))` otherwise `BigNumber` will fail, you can import `bn` from the `shared` subrepo (its a helper to create `BigNumber`s)

Reply:
Works fine for me, you can import this `term-id-getter.js` file in your local script with ganache node running.

Reply:
Based on previous experience with BigNumber, I insist that it's safer to use it that way, for consistency and to avoid having issues with different BigNumber versions as this dependency is not usually pinned in the libraries we use

Reply:
ok, added!